extensions/amp-megaphone/0.1/test/test-amp-megaphone.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../amp-megaphone';
+
+describes.realWin(
+  'amp-megaphone',
+  {
+    amp: {
+      extensions: ['amp-megaphone'],
+    },
+  },
+  env => {
+    const episodeEmbedUrl = 'https://player.megaphone.fm/OSC7749686951/';
+    const playlistEmbedUrl = 'https://playlist.megaphone.fm/?p=DEM6640968282';
+
+    let win, doc;
```

Comment:
Style guide says one variable declaration per line. A lot of the tests are not good about this.